### DISCUSSION TITLE
Change default ordering of Lucid Model

### DISCUSSION THREAD

**Discussion by zmaglica** [adonisjs/core#1272] [Help] [answered]
By default, when you execute `.all() ` Lucid Model function you get results from the database ordered by primary key in descending order. Is there a way to change data order by primary key in ascending order (or disable ordering at all)?

Regards.

**Comment by thetutlage** [comment, marked as answer]
Simply constructor a query yourself.

```ts
await Model.query().orderBy('key', 'direction')
```

**Reply by zmaglica** [reply]
Ah okay. I was thinking that there is a configuration key or flag to change the default order. Thank you for your help.

**Reply by thetutlage** [reply]
Nope, doesn't make sense to add configuration for these sort of things. There are many simple ways to achieve the expected outcome.

- One is to construct the query (as shown above)
- Another is to overwrite the `all` method on the model itself.
